barretenberg/acir_tests/sol-test/ZKHonkTest.sol

```diff
@@ -0,0 +1,18 @@
+// THIS FILE WILL NOT COMPILE BY ITSELF
+// Compilation is handled in `src/index.js` where solcjs gathers the dependencies
+
+pragma solidity >=0.8.4;
+
+import {HonkVerifier} from "./ZKVerifier.sol";
```

Comment:
This file isnt needed? cant we just name the generated verifier "./Verifier.sol" and it will just work? 

REVIEWNOTE: check that this ends up running in the same folder as the generated verifier

Reply:
if you run the honk zk and non-zk  tests in parallel (which is done in CI), unless you make separate files, you'll have concurrency issues